"2019-08-24 15:20:25,199 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 15:20:25,199 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 15:20:25,823 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 15:20:25,823 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 15:20:47,828 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 15:20:47,828 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 15:20:47,837 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 15:20:47,837 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 15:21:07,879 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 15:21:07,879 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 15:21:08,120 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 15:21:08,121 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 15:21:28,581 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 15:21:28,581 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 15:21:29,038 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 15:21:29,038 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 15:21:38,906 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 15:21:38,906 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 15:21:38,971 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 15:21:38,971 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-24 15:24:23,538 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824151653/3e3a008e-c683-11e9-a438-0242ac110002', but this can be ignored sometimes"
"2019-08-24 15:25:05,240 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-24 15:29:38,392 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824151653/fa5a05a2-c683-11e9-a438-0242ac110002', but this can be ignored sometimes"
"2019-08-24 15:33:12,595 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 39] Directory not empty: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824151653/79c7838c-c684-11e9-a438-0242ac110002', but this can be ignored sometimes"
"2019-08-24 15:33:33,250 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 39] Directory not empty: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824151653/85a4bf62-c684-11e9-a438-0242ac110002', but this can be ignored sometimes"
"2019-08-24 15:36:36,289 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 17] File exists: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824151653/f321b068-c684-11e9-a438-0242ac110002/0', but this can be ignored sometimes"
"2019-08-24 15:36:56,461 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190824151653/ff166814-c684-11e9-a438-0242ac110002', but this can be ignored sometimes"
